RxCocoa/Common/Observable+Bind.swift

```diff
@@ -10,82 +10,140 @@
+     writing more consistent binding code.
+
+     - parameter to: Observers to receives events.
+     - returns: Disposable object that can be used to unsubscribe the observers.
+     */
+    public func bind<O: ObserverType>(to observers: O...) -> Disposable where O.E == E {
+        return self.bind(to: observers)
     }
 
     /**
-     Creates new subscription and sends elements to observer.
+     Creates new subscription and sends elements to observer(s).
 
-     In this form it's equivalent to `subscribe` method, but it communicates intent better, and enables
+     In this form, it's equivalent to the `subscribe` method, but it better conveys intent, and enables
      writing more consistent binding code.
 
-     - parameter to: Observer that receives events.
+     - parameter to: Observers to receives events.
+     - returns: Disposable object that can be used to unsubscribe the observers.
+     */
+    public func bind<O: ObserverType>(to observers: O...) -> Disposable where O.E == E? {
+        return self.map { $0 as E? }.bind(to: observers)
+    }
+
+    /**
+     Creates new subscription and sends elements to observer(s).
+
+     In this form, it's equivalent to the `subscribe` method, but it better conveys intent, and enables
+     writing more consistent binding code.
+
+     - parameter to: Observers to receives events.
+     - returns: Disposable object that can be used to unsubscribe the observers.
+     */
+    private func bind<O: ObserverType>(to observers: [O]) -> Disposable where O.E == E {
+        return self.subscribe { event in
+            observers.forEach { $0.on(event) }
+        }
+    }
+
+    /**
+     Creates new subscription and sends elements to publish relay(s).
+
+     In case error occurs in debug mode, `fatalError` will be raised.
+     In case error occurs in release mode, `error` will be logged.
+
+     - parameter to: Target publish relays for sequence elements.
      - returns: Disposable object that can be used to unsubscribe the observer.
      */
-    public func bind<O: ObserverType>(to observer: O) -> Disposable where O.E == E? {
-        return self.map { $0 }.subscribe(observer)
+    public func bind(to relays: PublishRelay<E>...) -> Disposable {
+        return bind(to: relays)
     }
 
+
     /**
-     Creates new subscription and sends elements to publish relay.
+     Creates new subscription and sends elements to publish relay(s).
      
      In case error occurs in debug mode, `fatalError` will be raised.
      In case error occurs in release mode, `error` will be logged.
      
-     - parameter to: Target publish relay for sequence elements.
+     - parameter to: Target publish relays for sequence elements.
      - returns: Disposable object that can be used to unsubscribe the observer.
      */
-    public func bind(to relay: PublishRelay<E>) -> Disposable {
-        return self.subscribe { e in
+    public func bind(to relays: PublishRelay<E?>) -> Disposable {
```

Comment:
This doesn't look right to me. Not variadic.

Reply:
Yup, my bad :)